src/main/java/com/github/black0nion/blackonionbot/commands/misc/PollCommand.java

```diff
@@ -2,31 +2,419 @@
+								.addOption(OptionType.STRING, OPTION_FOUR, "The fourth option.", true)
+								.addOption(OptionType.STRING, OPTION_FIVE, "The fifth option.", true)
+								.addOption(OptionType.STRING, OPTION_SIX, "The sixth option.", true)
+								.addOption(OptionType.STRING, OPTION_SEVEN, "The seventh option.", true),
+						new SubcommandData(EIGHT_OPTIONS, "Used to create a poll with eight options.")
+								.addOption(OptionType.STRING, TITLE, "The title of the poll.", true)
+								.addOption(OptionType.STRING, OPTION_ONE, "The first option.", true)
+								.addOption(OptionType.STRING, OPTION_TWO, "The second option.", true)
+								.addOption(OptionType.STRING, OPTION_THREE, "The third option.", true)
+								.addOption(OptionType.STRING, OPTION_FOUR, "The fourth option.", true)
+								.addOption(OptionType.STRING, OPTION_FIVE, "The fifth option.", true)
+								.addOption(OptionType.STRING, OPTION_SIX, "The sixth option.", true)
+								.addOption(OptionType.STRING, OPTION_SEVEN, "The seventh option.", true)
+								.addOption(OptionType.STRING, OPTION_EIGHT, "The eighth option.", true),
+						new SubcommandData(NINE_OPTIONS, "Used to create a poll with nine options.")
+								.addOption(OptionType.STRING, TITLE, "The title of the poll.", true)
+								.addOption(OptionType.STRING, OPTION_ONE, "The first option.", true)
+								.addOption(OptionType.STRING, OPTION_TWO, "The second option.", true)
+								.addOption(OptionType.STRING, OPTION_THREE, "The third option.", true)
+								.addOption(OptionType.STRING, OPTION_FOUR, "The fourth option.", true)
+								.addOption(OptionType.STRING, OPTION_FIVE, "The fifth option.", true)
+								.addOption(OptionType.STRING, OPTION_SIX, "The sixth option.", true)
+								.addOption(OptionType.STRING, OPTION_SEVEN, "The seventh option.", true)
+								.addOption(OptionType.STRING, OPTION_EIGHT, "The eighth option.", true)
+								.addOption(OptionType.STRING, OPTION_NINE, "The ninth option.", true),
+						new SubcommandData(TEN_OPTIONS, "Used to create a poll with ten options.")
+								.addOption(OptionType.STRING, TITLE, "The title of the poll.", true)
+								.addOption(OptionType.STRING, OPTION_ONE, "The first option.", true)
+								.addOption(OptionType.STRING, OPTION_TWO, "The second option.", true)
+								.addOption(OptionType.STRING, OPTION_THREE, "The third option.", true)
+								.addOption(OptionType.STRING, OPTION_FOUR, "The fourth option.", true)
+								.addOption(OptionType.STRING, OPTION_FIVE, "The fifth option.", true)
+								.addOption(OptionType.STRING, OPTION_SIX, "The sixth option.", true)
+								.addOption(OptionType.STRING, OPTION_SEVEN, "The seventh option.", true)
+								.addOption(OptionType.STRING, OPTION_EIGHT, "The eighth option.", true)
+								.addOption(OptionType.STRING, OPTION_NINE, "The ninth option.", true)
+								.addOption(OptionType.STRING, OPTION_TEN, "The tenth option.", true)))
```

Comment:
What about optional options?

Reply:
what do you mean?

Reply:
You can just use a single subcommand and make all the options optional instead

Reply:
I thought about that but it is just too annoying and this is a better solution.

Reply:
"better"
Imo there's a lot of boilerplate in there, I'd rather have a shorter solution.
If you really want to use subcommands, use loops